src/server/controllers/user.js

```diff
@@ -0,0 +1,31 @@
+const { PrismaClient } = require('@prisma/client');
+const prisma = new PrismaClient();
+const express = require('express');
+const bcrypt = require('bcrypt');
+
+const saltRounds = 10;
+const secret = "kjfjksn"
+
+const postRegisterDetails = async (req, res) => {
+    try {
+        const { email, username, password } = req.body;
+        const hashedPassword = await bcrypt.hash(password, saltRounds);
```

Comment:
Auth with salt rounds! Perfect :)

Reply:
We can manage the salt rounds for performance better this way I feel.

Reply:
Definitely. I think 10 is a good number too. It'd be interesting to see what the typical number is